src/v/raft/types.h

```diff
@@ -522,14 +522,24 @@ enum class consistency_level { quorum_ack, leader_ack, no_ack };
 struct replicate_options {
     explicit replicate_options(consistency_level l)
       : consistency(l)
-      , timeout(std::nullopt) {}
+      , timeout(std::nullopt)
+      , _force_flush(false) {}
 
     replicate_options(consistency_level l, std::chrono::milliseconds timeout)
       : consistency(l)
-      , timeout(timeout) {}
+      , timeout(timeout)
+      , _force_flush(false) {}
+
+    // Callers may choose to force flush on an individual replicate request
+    // basis. This is useful if certain callers intend to override any
+    // default behavior at global/topic scope.
+    void set_force_flush() { _force_flush = true; }
```

Comment:
I'm debating whether to have a separate override like this or to make it a mandatory parameter in the constructor. Latter seems better because that forces the (future) developers to actually lookup what force flush is and not miss it, particularly when replicating new commands to the user topics.